### ISSUE TITLE
NTR: CT, MRI or angiography: bone vascular malformation (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8945]
New term request CT, MRI or angiography: bone vascular malformation (elthat_110121014659): AVM; arteriovenous malformation  UMLS: 1336946

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000587
- label: Bone arteriovenous malformation
- parents: [Arteriovenous malformation (HP:0100026)](https://hpo.jax.org/app/browse/term/HP:0100026)
- pmid: 
- def: An anomalous configuration of blood vessels that shunts arterial blood directly into veins without passing through the capillaries and that is located in a bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8945
- synonyms: Bone vascular malformation(NONE)
